lib/api/protocol/moveTo.js

```diff
@@ -25,25 +27,41 @@ module.exports = class Command extends ProtocolAction {
     return true;
   }
 
-  command(webElementId, xoffset, yoffset, callback = function() {}) {
-   
-    if (arguments.length >= 3 && typeof arguments[0] === 'string' || arguments[0] === null) {
+  command(...args) {
+    let webElementId;
+    let xoffset = 0;
+    let yoffset = 0;
+    let callback = function() {};
+
+    const lastItem = args[args.length - 1];
+    if (Utils.isFunction(lastItem)) {
+      args.pop();
+      callback = lastItem;
+    }
+
+    if (typeof args[0] === 'string' || arguments[0] === null) {
```

Comment:
We can use the methods on `Util` here . 

```suggestion
    if (Utils.isString(args[0])  || Utils.isNull(arguments[0])) {
```

Reply:
Used `isString` from `Utils`, but not `isNull` as it's undeclared.